downburst/template.py

```diff
@@ -92,6 +99,22 @@ def volume_clone(
     etree.SubElement(backing, 'path').text = parent_vol.key()
     return root
 
+def get_emulator_path():
+    global emulator_path
+    if emulator_path:
+        return emulator_path
+    log.debug('The host distro id is %s', distro.id())
+    if any(distro.id().startswith(_)
+                for _ in ('opensuse', 'sles')):
+        path = '/usr/bin/qemu-kvm'
+    elif any(distro.id().startswith(_)
+                for _ in ('centos', 'fedora', 'rhel')):
+        path = '/usr/libexec/qemu-kvm'
+    elif any(distro.id().startswith(_)
+                for _ in ('ubuntu', 'debian')):
+        path = '/usr/bin/kvm'
```

Comment:
might want to throw if none of the prefixes matches, just for the sake of completeness or error-proofing?